client/components/CustomVocabulariesFields.tsx

```diff
@@ -31,12 +31,9 @@ export default class CustomVocabulariesFields extends React.PureComponent<IProps
 
         const {
             errors,
-            diff,
         } = fieldProps;
 
-        const language = MAIN_LANGUAGE ?? get(diff, 'language') ?? getUserInterfaceLanguageFromCV();
-
-        console.log(diff, 'diff');
+        const language = this.props.language ?? getUserInterfaceLanguageFromCV();
```

Comment:
move `getUserInterfaceLanguageFromCV()` out of this file - apply it before passing language prop to this file